composable_kernel/include/utility/config.hpp

```diff
@@ -123,6 +123,11 @@
 #define CK_WORKAROUND_SWDEV_XXXXXX_INT8_DS_WRITE_ISSUE 1
 #endif
 
+// using split k xdlops kernel
+#ifndef CK_USE_SPLITK_XDLOPS
```

Comment:
Should not use Macro to enable split-K

Reply:
if original gridsize < 360, it will using split-k, else it will not using split-k. so split-k is compiled all the time.  The Macro is only for comparison with gridwise gemm v2r3

Reply:
done